.github/workflows/build.yaml

```diff
@@ -2,81 +2,141 @@ name: packages
 on:
   push:
     tags:
-    - 'v[0-9]+.[0-9]+.[0-9]+'
-    - 'v[0-9]+.[0-9]+.[0-9]+a[0-9]+'
-    - 'v[0-9]+.[0-9]+.[0-9]+b[0-9]+'
-    - 'v[0-9]+.[0-9]+.[0-9]+rc[0-9]+'
+      - "v[0-9]+.[0-9]+.[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+a[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+b[0-9]+"
+      - "v[0-9]+.[0-9]+.[0-9]+rc[0-9]+"
   # Dry-run only
   workflow_dispatch:
   schedule:
-    - cron: '0 13 * * SUN'
+    - cron: "0 13 * * SUN"
+
+defaults:
+  run:
+    shell: bash -e {0}
+
+env:
+  PYTHON_VERSION: "3.11"
+  PACKAGE: "param"
 
 jobs:
+  waiting_room:
```

Comment:
> I have limited it to lead maintainers (and me).

Ok. Where is the least of lead maintainers defined?

> I use it to double-check that the version tag is correct.

Too bad we no longer trust the system to produce the right version tag, but that's fine.

Reply:
> PyPi also strongly encourages it ([ref](https://docs.pypi.org/trusted-publishers/using-a-publisher/)).

I'm sorry I don't understand, this page links to using a trusted publisher. Are we doing that?

Reply:
> I'm sorry I don't understand, this page links to using a trusted publisher. Are we doing that?

Just to be sure: 

- The waiting room job itself is to ensure that all the building steps succeed. For example, if the PyPi build step fails, the conda package will _not_ be released. 
- Inside the waiting room is an environment `publish`, which people must be approved before release. This was what I tried to reference in the PyPI ref.
![image](https://github.com/user-attachments/assets/fd0573ec-afec-4b40-8dda-dd6cceff4d50)

Reply:
Ok thanks I got it.

> The waiting room job itself is to ensure that all the building steps succeed. For example, if the PyPi build step fails, the conda package will not be released.

It sounds like it's more for manually checking the two builds, since otherwise it could be automated.

> Inside the waiting room is an environment publish, which people must be approved before release. This was what I tried to reference in the PyPI ref.

Ok I understand better how you use the environment to add this protection rule. Seems not so related to the trusted publisher thing for which `environment` is optional but `permissions` with `id-token: write` is not.

![image](https://github.com/user-attachments/assets/d12b866b-9550-4493-821a-63c5e40a7ff2)

By the way I think you should add Jean-Luc and we probably need to update the list of maintainers.

![image](https://github.com/user-attachments/assets/826d95d5-cff8-47bd-b5b2-3965d77a428d)

All of that sounds like it would be worth being documented somewhere, it's kind of hard to follow.

Reply:
> manually checking the two builds

It is automatically done. If the environment were not there, it would do everything without interaction. 

> All of that sounds like it would be worth being documented somewhere

Yes. I will update the holoviz site (soon).